src/app/invitation-message/indicator/invitation-message-indicator.run.js

```diff
@@ -0,0 +1,19 @@
+const _ = require('lodash');
```

Comment:
DO you still need dynamic directive?

Reply:
I understand your point, in my case, the inject condition is quite complicated
```
function runBlock(dynamicDirectiveService, DynamicDirective, INVITATION_MESSAGE_HEADERS) {
    var shouldInject = function(scope) {
      return scope.email &&
        scope.email.headers &&
        INVITATION_MESSAGE_HEADERS.UID in scope.email.headers;
    };
    var directive = new DynamicDirective(shouldInject, 'cal-inbox-invitation-message-blue-bar', {
      attributes: [{ name: 'message', value: 'email' }]
    });

    dynamicDirectiveService.addInjection('inbox-message-info', directive);
```
So, it it ok if I keep it like this for now.

Reply:
I do not get it, didn't we said goodbye to dynamic directives?

Reply:
What Tuan say is that dynamic directives provides a way to choose whether we display, or not, the widget.
Here it's:
```
return scope.email &&
        scope.email.headers &&
        INVITATION_MESSAGE_HEADERS.UID in scope.email.headers;
```
If we ask developers to switch away from dynamic directives, Christophe can you give to the developers the mthod to choose component display ?
Maybe something like: 
```
$scope.mivisible = function() {
      return $scope.email &&
        $scope.email.headers &&
        INVITATION_MESSAGE_HEADERS.UID in $scope.email.headers;
    };
<inbox-message-info ng-if='mevisible()'>
```

Please Christophe provide tech guidance here. Tks

Reply:
I removed the dynamic directive